frontend/src/components/Contact/Contact.jsx

```diff
@@ -68,20 +78,24 @@ const Contact = () => {
           legend="Je suis"
           isInline
           required
+          value={user}
```

Comment:
This is not working, I don't know why..

Reply:
Because when the page is loading, user is ' '.
You have to put conditions, like when the user is loaded , then you can show the form.

Reply:
You have an exemple in the branch ' modif/input-react-dsfr ' in ' ConventionFrom.jsx'

Reply:
`user` is the value of the field (not a response from a request) and it is set with location. It's in this case that it doesn't work as it should.

Reply:
Did you try to put a default value ? and not a ' ' ?

Reply:
Thanks @Evidorus, it works with default value set properly :)